README.md

```diff
@@ -28,6 +28,22 @@ The site is published automatically by [Netlify](https://www.netlify.com/) whene
 
 We strongly encourage you to contribute to this site! For more information, see the [contributing](CONTRIBUTING.md) guide.
 
+### Publishing new Jaeger version
+
+Each Jaeger version is documented in a separate directory e.g. [content/docs/1.8](./content/docs//1.8).
+
+When submitting changes for a new version of Jaeger, please first make a copy
+of the current version's documentation under the new version number and commit it.
+Then apply any relevant changes to update the documentation for the new version
+as a separate PR, to make it easier to review the changes. A possible workflow is
+
+1. copy files from old version to a new directory
+2. add the new version to [config.toml](./config.toml)
```

Comment:
There are versions vere for the downloads page, but iirc the content pages do not require config changes to be rendered. I don't recall if the latest link will upgrade automatically, maybe we should make it explicit, then copying can be done as independent PR.

One downside of this current process is that once the PR is merged it will still show changes and copying in a single commit (unless we re-enable unsquashed merges). I'd rather have two PRs.